### ISSUE TITLE
[Submission] Raytrax

### ISSUE THREAD

**Issue by DavidMStraub** [pysimhub/pysimhub.github.io#122]
### Project Name

Raytrax

### Tagline

 Electron Cyclotron Resonance Heating for Fusion Plasmas in JAX

### Description

Raytrax simulates Electron Cyclotron Resonance Heating (ECRH) of magnetic confinement fusion plasmas. Powered by [JAX](https://docs.jax.dev/), it features JIT-compiled ray tracing and automatic differentiation – making it well-suited for gradient-based beam optimization in fusion plant design.

### Select all tags that apply

- [ ] block-diagram
- [ ] calibration
- [ ] circuit
- [ ] computational neuroscience
- [ ] control
- [ ] crowd management
- [ ] crowd simulation
- [ ] cycle simulation
- [ ] des
- [ ] digital-twin
- [ ] dynamic-systems
- [x] electromagnetic
- [ ] emulation
- [ ] engineering
- [ ] evacuation
- [ ] finite-element
- [x] heating
- [ ] homogenization
- [ ] hydrogen
- [ ] microwave
- [ ] mujoco
- [ ] network
- [ ] neuroscience
- [ ] nonlinear
- [ ] pedestrian dynamics
- [x] physics
- [x] powerplant
- [ ] quaternions
- [ ] radio-frequency
- [ ] refrigeration
- [ ] rf
- [ ] robotics
- [ ] sensitivity analysis
- [x] simulation
- [ ] snn
- [ ] spiking neural network
- [ ] system-modeling
- [ ] thermodynamics
- [ ] transient
- [ ] translation
- [ ] uncertainty quantification
- [ ] vensim
- [ ] vibration
- [ ] visualization
- [ ] xmile

### Additional Tags

fusion

### Repository

https://github.com/proximafusion/raytrax

### Documentation

https://proximafusion.github.io/raytrax

### PyPI

https://pypi.org/project/raytrax/

### Example or Tutorial

https://proximafusion.github.io/raytrax/getting-started/

### Logo

https://proximafusion.github.io/raytrax/assets/proxima.svg

**Comment by DavidMStraub** [comment]
Fixed.
